keyboards/converter/ps2_usb/keymaps/default/keymap.c

```diff
@@ -0,0 +1,46 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
```

Comment:
`#include QMK_KEYBOARD_H` includes `action_layer.h` automatically at build time. This line can be safely deleted.

```suggestion
```